### PR TITLE
Fix crash on NVIDIA 2Gb GPU.

### LINKED ISSUE #201: Crash wiht high texture quality on load using NVIDIA graphics card
**Describe the Bug**
Webots crashes loading worlds if the Texture Quality is set to High.
My NVIDIA graphics card is not detected as low-gpu and by default the texture quality is set to high.

**Steps to Reproduce**
1. Install Webots / reset Webots configurations
2. Go to 'webots_box.wbt'

**System**
 - Operating System: Windows 10
 - Graphics Card: NVIDIA GeForce GT 640M with 2GB dedicated VRAM

**Stack**
```
Thread 14 received signal ?, Unknown signal.
[Switching to Thread 7080.0x1b54]
0x000000001eef9389 in vk_optimusGetInstanceProcAddr ()
   from C:\WINDOWS\System32\DriverStore\FileRepository\nvaci.inf_amd64_cbe7e13da3856715\nvoglv64.dll
(gdb) bt
#0  0x000000001eef9389 in vk_optimusGetInstanceProcAddr ()
   from C:\WINDOWS\System32\DriverStore\FileRepository\nvaci.inf_amd64_cbe7e13da3856715\nvoglv64.dll
#1  0x000000001eafe7b3 in nvoglv64!DrvValidateVersion ()
   from C:\WINDOWS\System32\DriverStore\FileRepository\nvaci.inf_amd64_cbe7e13da3856715\nvoglv64.dll
#2  0x000000001eb4e99f in nvoglv64!DrvPresentBuffers ()
   from C:\WINDOWS\System32\DriverStore\FileRepository\nvaci.inf_amd64_cbe7e13da3856715\nvoglv64.dll
#3  0x000000001eb4cd86 in nvoglv64!DrvPresentBuffers ()
   from C:\WINDOWS\System32\DriverStore\FileRepository\nvaci.inf_amd64_cbe7e13da3856715\nvoglv64.dll
#4  0x000000001eb3ad56 in nvoglv64!DrvPresentBuffers ()
   from C:\WINDOWS\System32\DriverStore\FileRepository\nvaci.inf_amd64_cbe7e13da3856715\nvoglv64.dll
#5  0x000000001ec4395e in nvoglv64!DrvPresentBuffers ()
   from C:\WINDOWS\System32\DriverStore\FileRepository\nvaci.inf_amd64_cbe7e13da3856715\nvoglv64.dll
#6  0x000000001ec46714 in nvoglv64!DrvPresentBuffers ()
   from C:\WINDOWS\System32\DriverStore\FileRepository\nvaci.inf_amd64_cbe7e13da3856715\nvoglv64.dll
#7  0x000000001ec459ed in nvoglv64!DrvPresentBuffers ()
   from C:\WINDOWS\System32\DriverStore\FileRepository\nvaci.inf_amd64_cbe7e13da3856715\nvoglv64.dll
#8  0x000000001ec46c1b in nvoglv64!DrvPresentBuffers ()
   from C:\WINDOWS\System32\DriverStore\FileRepository\nvaci.inf_amd64_cbe7e13da3856715\nvoglv64.dll
#9  0x000000001ec5e029 in nvoglv64!DrvPresentBuffers ()
   from C:\WINDOWS\System32\DriverStore\FileRepository\nvaci.inf_amd64_cbe7e13da3856715\nvoglv64.dll
#10 0x000000001eafb94e in nvoglv64!DrvValidateVersion ()
   from C:\WINDOWS\System32\DriverStore\FileRepository\nvaci.inf_amd64_cbe7e13da3856715\nvoglv64.dll
#11 0x000000001ec1fe52 in nvoglv64!DrvPresentBuffers ()
   from C:\WINDOWS\System32\DriverStore\FileRepository\nvaci.inf_amd64_cbe7e13da3856715\nvoglv64.dll
#12 0x000000001ebe3fc6 in nvoglv64!DrvPresentBuffers ()
   from C:\WINDOWS\System32\DriverStore\FileRepository\nvaci.inf_amd64_cbe7e13da3856715\nvoglv64.dll
#13 0x000000001ec44327 in nvoglv64!DrvPresentBuffers ()
   from C:\WINDOWS\System32\DriverStore\FileRepository\nvaci.inf_amd64_cbe7e13da3856715\nvoglv64.dll
#14 0x000000001ec41a47 in nvoglv64!DrvPresentBuffers ()
   from C:\WINDOWS\System32\DriverStore\FileRepository\nvaci.inf_amd64_cbe7e13da3856715\nvoglv64.dll
#15 0x000000001ec422b1 in nvoglv64!DrvPresentBuffers ()
   from C:\WINDOWS\System32\DriverStore\FileRepository\nvaci.inf_amd64_cbe7e13da3856715\nvoglv64.dll
#16 0x000000001ec0af37 in nvoglv64!DrvPresentBuffers ()
   from C:\WINDOWS\System32\DriverStore\FileRepository\nvaci.inf_amd64_cbe7e13da3856715\nvoglv64.dll
#17 0x000000001ec16d44 in nvoglv64!DrvPresentBuffers ()
   from C:\WINDOWS\System32\DriverStore\FileRepository\nvaci.inf_amd64_cbe7e13da3856715\nvoglv64.dll
#18 0x000000001ec0ac93 in nvoglv64!DrvPresentBuffers ()
   from C:\WINDOWS\System32\DriverStore\FileRepository\nvaci.inf_amd64_cbe7e13da3856715\nvoglv64.dll
#19 0x000000001ec18dab in nvoglv64!DrvPresentBuffers ()
   from C:\WINDOWS\System32\DriverStore\FileRepository\nvaci.inf_amd64_cbe7e13da3856715\nvoglv64.dll
#20 0x000000001ebb3e83 in nvoglv64!DrvPresentBuffers ()
   from C:\WINDOWS\System32\DriverStore\FileRepository\nvaci.inf_amd64_cbe7e13da3856715\nvoglv64.dll
#21 0x000000001ec13189 in nvoglv64!DrvPresentBuffers ()
   from C:\WINDOWS\System32\DriverStore\FileRepository\nvaci.inf_amd64_cbe7e13da3856715\nvoglv64.dll
#22 0x000000001ec09382 in nvoglv64!DrvPresentBuffers ()
   from C:\WINDOWS\System32\DriverStore\FileRepository\nvaci.inf_amd64_cbe7e13da3856715\nvoglv64.dll
#23 0x000000001ec115bd in nvoglv64!DrvPresentBuffers ()
   from C:\WINDOWS\System32\DriverStore\FileRepository\nvaci.inf_amd64_cbe7e13da3856715\nvoglv64.dll
#24 0x000000001ec133dc in nvoglv64!DrvPresentBuffers ()
   from C:\WINDOWS\System32\DriverStore\FileRepository\nvaci.inf_amd64_cbe7e13da3856715\nvoglv64.dll
#25 0x000000001e871a43 in ?? ()
   from C:\WINDOWS\System32\DriverStore\FileRepository\nvaci.inf_amd64_cbe7e13da3856715\nvoglv64.dll
#26 0x000000001e88042a in ?? ()
   from C:\WINDOWS\System32\DriverStore\FileRepository\nvaci.inf_amd64_cbe7e13da3856715\nvoglv64.dll
#27 0x000000001e71d66d in ?? ()
   from C:\WINDOWS\System32\DriverStore\FileRepository\nvaci.inf_amd64_cbe7e13da3856715\nvoglv64.dll
#28 0x000000001e8b1e70 in ?? ()
   from C:\WINDOWS\System32\DriverStore\FileRepository\nvaci.inf_amd64_cbe7e13da3856715\nvoglv64.dll
#29 0x000000001e8b1c20 in ?? ()
   from C:\WINDOWS\System32\DriverStore\FileRepository\nvaci.inf_amd64_cbe7e13da3856715\nvoglv64.dll
#30 0x000000001eb00d6b in nvoglv64!DrvValidateVersion ()
   from C:\WINDOWS\System32\DriverStore\FileRepository\nvaci.inf_amd64_cbe7e13da3856715\nvoglv64.dll
#31 0x00007ffcda653034 in KERNEL32!BaseThreadInitThunk ()
   from C:\WINDOWS\System32\kernel32.dll
#32 0x00007ffcdc783691 in ntdll!RtlUserThreadStart ()
   from C:\WINDOWS\SYSTEM32\ntdll.dll
#33 0x0000000000000000 in ?? ()
Backtrace stopped: previous frame inner to this frame (corrupt stack?)
```

### PR DESCRIPTION
**Pull request by DavidMansolino**
Fix #201

### CODE REVIEW

**Comment by DavidMansolino** [comment]
test me

**Comment by DavidMansolino** [comment]
:+1:

**Comment by stefaniapedrazzi** [comment]
:eyes:

**Comment by stefaniapedrazzi** [comment]
I fixed a typo directly. I still have to check of to add suggestion changes.

**Comment by stefaniapedrazzi** [comment]
I have some issues with the nvidia graphics card after updating the drivers. I'm reinstalling them but it could take some time.

File: src/webots/nodes/WbCubemap.cpp [hunk 1]
```diff
@@ -240,8 +240,11 @@ void WbCubemap::loadWrenTexture() {
 
   const int quality = WbPreferences::instance()->value("OpenGL/textureQuality", 2).toInt();
   // maps the quality eihter to '0: 64, 1: 128, 2: 256' or in case of HDR to '0: 32, 1: 64, 2: 256'
-  const int offset = (mIsEquirectangular && quality < 2) ? 5 : 6;
-  const int resolution = pow(2, offset + quality);  // 0: 64, 1: 128, 2: 256
+  const int offset =
+    (mIsEquirectangular && (quality < 2 || WbPreferences::instance()->value("OpenGL/limitBakingResolution", false).toBool())) ?
+      5 :
+      6;
+  const int resolution = pow(2, offset + quality);
```
Comment (omichel) [inline src/webots/nodes/WbCubemap.cpp]:
```suggestion
  const int resolution = 1 << (offset + quality);
```

**Pull request merged**